subcommittees/mentoring/Micro-Lesson_Implementation.md

```diff
@@ -0,0 +1,61 @@
+## Implementation Support Plan for Micro-lesson
+
+Below is an implementation support plan for the proposed Micro-Lesson to be taught by 
+those who have recently completed instructor training but not yet taught a Software 
+Carpentry workshop. After successfully completing the workflow outlined below, the student
+can then graduate and become a badged Software Carpentry instructor.
+
+
+Estimating an average of 40 people completing instructor training, and that all of them
```

Comment:
From data that @gvwilson provide me we had 285 only in the first half of this year. We need to rethink the numbers.

As we discussed on June 12th, maybe we could run this four times/year (once for each season).